crates/rv/src/commands/ci.rs

```diff
@@ -639,9 +674,13 @@ where
     })
 }
 
-fn convert_gemspec_yaml_to_ruby(contents: String) -> String {
-    let mut child = std::process::Command::new("ruby")
+fn convert_gemspec_yaml_to_ruby(contents: String, ruby_version: &RubyRequest) -> String {
+    let mut child = std::process::Command::new("rv")
         .args([
+            "ruby",
+            "run",
+            &ruby_version.to_string(),
+            "--",
             "-e",
             "Gem.load_yaml; print Gem::SafeYAML.safe_load(ARGF.read).to_ruby",
         ])
```

Comment:
Just a Rust question since I'm new: would it make sense to extract a Runner struct that we inject the ruby_version into and then do this?

```rust
pub fn run(&self, args) {
  std::process::Command::new("rv").args(["ruby", "run", &self.ruby_version.to_string(), "--", ..args]).output()
}
```

then we can pass the runner around instead of the plain `ruby_version`.